v2/tools/generator/internal/codegen/pipeline/json_serialization_test_cases.go

```diff
@@ -104,5 +103,13 @@ func (s *objectSerializationTestCaseFactory) AddTestTo(def astmodel.TypeDefiniti
 	isOneOf := astmodel.OneOfFlag.IsOn(def.Type()) // this is ugly but can’t do much better right now
 
 	testcase := testcases.NewJSONSerializationTestCase(def.Name(), container, isOneOf, s.idFactory)
+	if _, isResource := astmodel.AsResourceType(def.Type()); isResource {
+		// Don't need to test resources many times, the spec and status types are tested independently
+		testcase.SetMinSuccessfulTests(20)
```

Comment:
are 20 and 80 below arbitrary numbers? or is there such a reason to use these specific numbers?

Reply:
Relatively arbitrary - default is to run 100 trials, so 20xResource + 80xSped/Status adds to the default; Other than that, wanted to heavily bias towards testing Spec/Status in isolation.

Reply:
Makes sense! Not sure if we should add a comment mentioning that?

Reply:
I've updated the code to use some constants.